src/components/Tooltip.tsx

```diff
@@ -34,9 +33,14 @@ const Tooltip = ({
     <View style={styles.tooltipContainer}>
       <View style={styles.tooltip}>
         <View style={{ flexDirection: 'row', alignItems: 'flex-start' }}>
-          {iconName && (
+          {icon && (
             <View style={{ ...styles.iconContainer, marginRight: theme.margin.common }}>
-              <Icon name={iconName} size={theme.icon.size.extraLarge} />
+              <Icon
```

Comment:
Volgens mij is  `<Icon size={theme.icon.size.extraLarge} {...icon}>` ook een optie.

Reply:
👍🏻 Ik had al het gevoel dat dit beter kon. Aangepast, ook in `IconButton`.